tests/e2e/commandse2e/command.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
```

Comment:
I should more this function into e2e/utils/, and make test/e2e/commandse2e/ to just contain the testcases

Reply:
addressed